elements/pf-tabs/BaseTabs.ts

```diff
@@ -1,16 +1,19 @@
 import { LitElement, html } from 'lit';
 import { property, query, queryAssignedElements } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
 
 import { RovingTabindexController } from '@patternfly/pfe-core/controllers/roving-tabindex-controller.js';
-import { getRandomId } from '@patternfly/pfe-core/functions/random.js';
+import { OverflowController } from '@patternfly/pfe-core/controllers/overflow-controller.js';
```

Comment:
If we are making the overflow buttons optional, should/could we just dynamically import the controller only if the buttons will be used?

Reply:
we need to add element-internals-polyfill to the importMap, and we need to make sure to list all our polyfills in our comms to other teams

Reply:
https://github.com/patternfly/patternfly-elements/pull/2375/commits/b22840d0bb1d7db6a1f98c8a98a1f145e198ae75 should fix this

Reply:
it's not a bad idea to lazy-load the controller, but that can come afterwards in a later optimization, IMO.

I'd rather work on getting to the point where we can deprecate `BaseTab*` altogether, and just have pf-tab* and rh-tab* implement the same controllers